Driver_LED.dcm

```diff
@@ -192,6 +192,18 @@ K PWM LED driver QFN
 F http://www.ti.com/lit/ds/symlink/tlc5947.pdf
 $ENDCMP
 #
+$CMP TLC5971PWP
+D 12-Channel, 16-Bit, Enhanced Spectrum, PWM, RGB, LED Driver With 3.3-V Linear Regulator, HTSSOP-20
+K led driver
+F http://www.ti.com/lit/ds/symlink/tlc5971.pdf
+$ENDCMP
+#
+$CMP TLC5971RGE
+D 12-Channel, 16-Bit, Enhanced Spectrum, PWM, RGB, LED Driver With 3.3-V Linear Regulator, WQFN-24
```

Comment:
originally i think i read somewhere that its the same and kicad sticks to the W variant?!
but i also think its easier to use / search for if its in conformance with the datasheet

Reply:
Not sure! Could be, but I'm unsure. Because the footprint isn't standard, I think this is less of an issue. I would normally default to recommending that the description match the footprint used.